dev_cmds/lint.ts

```diff
@@ -0,0 +1,15 @@
+import { execPiped, runAsyncMain } from "devcmd";
+import { YARN_COMMAND } from "./utils/commands";
+import { repoRoot } from "./utils/paths";
+
+async function main() {
+  await execPiped({
+    command: YARN_COMMAND,
+    args: ["prettier", "--check", "."],
```

Comment:
hmmm, does this `.` refer to the repo root or the `dev_cmds` dir (in which all devcmd tasks are run)?

Reply:
Since the process is started in the repoRoot (check line 10) the . refers to the repo root ;-)

Reply:
ah, I had missed that :D then this looks good to go :)